backend/Services/DatasetService.cs

```diff
@@ -285,6 +286,7 @@ public async Task<DatasetResponse> DeleteAsync(int id)
             {
                 _datasetRepository.Remove(existingDataset);
                 await _unitOfWork.CompleteAsync();
+                // TODO: slett datasett fra gitlab??
```

Comment:
Hadde ikke vært dumt det nei, men frontenden har heller ikke mulighet for å slette datasett. Kunden sa vel selv at det ikke var prioritert.

Reply:
den kan stå inntil videre